### PR TITLE
Implements Homebrew Report And Google Analytics

### PR DESCRIPTION
**Pull request by CharlesJonah**
**Demo**
[https://www.useloom.com/share/a38ee0e765714b60ae2a13eaf3f9acf2](url)

### CODE REVIEW

File: blog/.gitignore [hunk 1]
```diff
@@ -12,6 +12,7 @@ yarn-error.log
 # Build directory
 /public
 .DS_Store
+../.DS_Store
```
Comment (adongu) [inline blog/.gitignore]: We can add a .gitignore on the outer folder to cover the ignores of outside folder instead.

File: blog/gatsby-browser.js [hunk 2]
```diff
@@ -3,5 +3,12 @@
  *
  * See: https://www.gatsbyjs.org/docs/browser-apis/
  */
+import ReactGA from "react-ga";
+import { wrapWithProvider } from "./src/index";
 
-// You can delete this file if you're not using it
+export const wrapRootElement = wrapWithProvider;
+
+export const onRouteUpdate = ({ location }) => {
```
Comment (adongu) [inline blog/gatsby-browser.js]: There doesn't seem to be any file that's importing this, how is this being used?

File: blog/.gitignore (hunk 1 above)
Comment (CharlesJonah) [reply blog/.gitignore]: that's correct

File: blog/gatsby-browser.js (hunk 2 above)
Comment (CharlesJonah) [reply blog/gatsby-browser.js]: This is used to by Provider to wrap all the elements with redux [store,](https://www.gatsbyjs.org/docs/ssr-apis/#wrapRootElement) and also onRouteUpdate, is used to track route changes which are being used by google [anaytics.](https://www.gatsbyjs.org/docs/browser-apis/#onRouteUpdate)

**Pull request merged**